examples/word-count/.template/pyproject.toml

```diff
@@ -1,9 +1,13 @@
 [build-system]
-requires = ["setuptools>=41.0.0", "wheel", "setuptools_rust>=1.0.0"]
+requires = ["maturin>=1,<2"]
```

Comment:
This does bring our `setuptools_rust` based examples down to one (the starter), right? Do we want this because Maturin is the preferred approach? (It would make sense to me, I would just like it being called out as a deliberate decision.)

Reply:
Actually this was already building with maturin in the noxfile, so I just fixed the inconsistency. But yes I think in general maturin is easier so it's been nicer to use it for examples.